server/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java

```diff
@@ -359,8 +360,18 @@ public final XContentBuilder toXContent(XContentBuilder builder, Params params)
             builder.startObject(getName());
         }
         if (this.metadata != null) {
-            builder.field(CommonFields.META.getPreferredName());
-            builder.map(this.metadata);
+            if (this.metadata.containsKey("exclude_deleted_docs")) {
```

Comment:
Nitpick: Can we extract `exclude_deleted_docs` into a constant?

Reply:
https://github.com/elastic/elasticsearch/pull/109494/commits/468476e924ea03241beb94289f5a53d69b226164